### DISCUSSION TITLE
제페토 내 오브젝트는 서로 공유가 되는지?

### DISCUSSION THREAD

**Discussion by JeongYire** [naverz/zepeto-studio-kor#291] [5. Q&A]
예를들어 오브젝트를 한번 밟으면 그 이후로 나타나지 않는다고 한다면.
그 오브젝트를 밟아서 삭제한 이후 다른사람이 제 방을 통해 들어온다면

그 오브젝트는 삭제된 상태인가요? 아니면 삭제되지 않은상태인가요?
이런 오브젝트의 변동은 서버단에서 저장을 해야되는건지 궁금합니다.

**Comment by x4cf** [comment]
삭제되지 않은 상태입니다.
오브젝트 변동은 서버와의 통신을 통해 해결해야합니다
